docs/releases/2.13.rst

```diff
@@ -95,3 +102,16 @@ Setting menu items now use SVG icons by default. For sites reusing built-in Wagt
         return SettingMenuItem(CustomSetting, icon='custom-cog')
         # After:
         return SettingMenuItem(CustomSetting, icon='', classnames='icon icon-custom-cog')
+
+
+CommentPanel
+~~~~~~~~~~~~
+
+``Page.content_panels`` now includes `CommentPanel`, which is used to save and load comments. If you are overriding page edit handlers and
+want to use the new commenting system, your list of edit handlers should be updated to include ``CommentPanel``.
+
+New data attribute in rich text
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To allow paragraphs in rich text to be identified consistently by the commenting system, the unique block key used in the Draftail editor
+for each block level element is now saved in the generated database html in the ``data-block-key`` attribute.
```

Comment:
Who does this affect, and what action do they need to take?

Reply:
This is one I wasn't sure about whether to include. It affects everyone using Draftail, but I can't really think of any circumstances under which it would break anything - it's more just a heads up that there will be an extra attribute in the html. Maybe it makes more sense just to remove this note?

Reply:
Removed the note for now.